fingerprint2.js

```diff
@@ -492,7 +496,7 @@
     },
     touchSupportKey: function (keys) {
       if(!this.options.excludeTouchSupport){
-        keys.push(this.getTouchSupport());
+        keys.push({key: "getTouchSupport", value: this.getTouchSupport()});
```

Comment:
Please also make more descriptive commit messages. "change fingerprint.js" is a really bad commit message. It says nothing.